Resources/Prototypes/_White/Entities/Objects/Misc/implanters.yml

```
@@ -0,0 +1,12 @@
+- type: entity
+  parent: Implanter
+  id: ImplanterSyndi
+  description: A compact disposable syringe exclusively designed for the quick injection and extraction of subdermal implants.
+  components:
+    - type: Item
+      sprite: Objects/Specific/Medical/syndi_implanter.rsi
+    - type: Sprite
+      sprite: Objects/Specific/Medical/syndi_implanter.rsi
+    - type: Implanter
+      drawTime: 2
+      implantTime: 2
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)